wrappers/python/simtk/openmm/app/forcefield.py

```diff
@@ -888,7 +888,23 @@ def _getResidueTemplateMatches(self, res, bondedToAtom, templateSignatures=None,
                 template = allMatches[0][0]
                 matches = allMatches[0][1]
             elif len(allMatches) > 1:
-                raise Exception('Multiple matching templates found for residue %d (%s): %s.' % (res.index+1, res.name, ', '.join(match[0].name for match in allMatches)))
+                # We found multiple matches.  This is OK if and only if they assign identical types and parameters to all atoms.
+                t1, m1 = allMatches[0]
```

Comment:
I worry that putting the logic for comparing templates here could cause issues if we neglect to update this when updating the information carried in `_TemplateData`? What about moving the method to check if the templates are identical to a method within the [`_TemplateData` class](https://github.com/openmm/openmm/blob/d88ddb3b7ac4bda2d636daaee4b0572bae9f6b25/wrappers/python/simtk/openmm/app/forcefield.py#L587)?